### ISSUE TITLE
Merging Accounts when Addresses share a Signer 

### ISSUE THREAD

**Issue by zakhap** [hicommonwealth/commonwealth#5080]
## Description

Two paths here:
1. Automerge accounts, impact is small enough
2. UX solution is required, blocking #5078.

**Comment by mhagel** [comment]
>I believe this can be deprecated? @mhagel

Yes, this is a duplicate of https://github.com/hicommonwealth/commonwealth/issues/5226 so this ticket can be closed.

We have not merged the accounts yet. All new addresses are automatically merged, but there are still some legacy addresses out there. 

@zakhap
